usr/src/cmd/zdb/zdb.c

```diff
@@ -3530,14 +3686,14 @@ main(int argc, char **argv)
 	nvlist_t *policy = NULL;
 	uint64_t max_txg = UINT64_MAX;
 	int rewind = ZPOOL_NEVER_REWIND;
+	const char *flags = "bcdehilmop:r:st:uvx:ACDFI:LMPRSU:X";
```

Comment:
% irb
irb(main):001:0> flags="bcdehilmop:r:st:uvx:ACDFI:LMPRSU:X".gsub(":","").split("").sort
=> ["A", "C", "D", "F", "I", "L", "M", "P", "R", "S", "U", "X", "b", "c", "d", "e", "h", "i", "l", "m", "o", "p", "r", "s", "t", "u", "v", "x"]

irb(main):002:0> oldflags="bcdhilmMI:suCDRSAFLXx:evp:t:U:P".gsub(":","").split("").sort
=> ["A", "C", "D", "F", "I", "L", "M", "P", "R", "S", "U", "X", "b", "c", "d", "e", "h", "i", "l", "m", "p", "s", "t", "u", "v", "x"]

irb(main):003:0> flags - oldflags
=> ["o", "r"]

Oops -- although there are no missing flags, there is an extra flag in that string.  (Another option I added locally... :)
I'll push up another commit to fix this, but this shouldn't affect the @zettabot testing being done.